python-checks-testkit/pom.xml

```diff
@@ -38,6 +38,11 @@
       <artifactId>assertj-core</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.sonarlint.core</groupId>
+      <artifactId>sonarlint-plugin-api</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
We probably don't need to a change to this file for the purpose of the current PR.